Sources/XCBuildSupport/XcodeBuildSystem.swift

```diff
@@ -52,7 +52,7 @@ public final class XcodeBuildSystem: BuildSystem {
     public init(
         buildParameters: BuildParameters,
         packageGraphLoader: @escaping () throws -> PackageGraph,
-        isVerbose: Bool,
+        isVerbose: Bool = false,
```

Comment:
as in the other spot, I prefer passing the full Verbosity value and not having a default so call-sites make conscious decision